pallets/anchor/src/lib.rs

```diff
@@ -342,8 +343,7 @@ impl<T: Config<I>, I: 'static> AnchorInspector<T, I> for Pallet<T, I> {
 		Ok(true)
 	}
 
-	/// Check if this anchor has this edge
 	fn has_edge(id: T::TreeId, src_chain_id: T::ChainId) -> bool {
-		EdgeList::<T, I>::contains_key(id, src_chain_id)
+		true
```

Comment:
Why?

Reply:
I guess this happened when I merged the branch.

Will fix that.